scripts/install.sh

```diff
@@ -11,7 +11,7 @@ BINARY_DOWNLOAD_PREFIX="https://github.com/apollographql/router/releases/downloa
 
 # Router version defined in apollo-router's Cargo.toml
 # Note: Change this line manually during the release steps.
-PACKAGE_VERSION="v1.28.0"
+PACKAGE_VERSION="v1.27.0"
```

Comment:
This shouldn't be changed.  We use `latest` to choose the install version.

```suggestion
PACKAGE_VERSION="v1.28.0"
```